elizhahn/Flash-Cards
# Refactor/clean up files

## Is this a fix or a feature?
- fix
## What is the change?
## What does it fix?
- cleans up the files
- adds the package-lock.json changes with npm install
## Where should the reviewer start
- util.js
- Round.js
## How should this be tested?
- run node index.js all functionality should be working
- run npm test, all tests should be passing
